core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserCoreConstants.java

```diff
@@ -114,6 +114,8 @@
     public static final String USER_LOCKED = "true";
     public static final String USER_UNLOCKED = "false";
 
+    public static final int SUPER_TENANT_ID = -1234;
```

Comment:
Isn't there an existing constant for this ? May be in multi tenant related repos ?